src/components/card/card.scss

```diff
@@ -112,6 +112,11 @@ $_focus-overflow: (size.$edge-large * -1);
     grid-template-columns: repeat(3, 1fr); /* 2 */
     grid-template-rows: 1fr repeat(3, minmax(0, auto)) 1fr; /* 3 */
 
+    .c-card__cover {
+      block-size: 100%; /* 4 */
+      inline-size: 100%; /* 4 */
+    }
```

Comment:
These styles should be closer to the element they are modifying, not nested within a modifier of the parent element (see previous thread).

Reply:
@tylersticka Got it!

So, in this case, it'd be something like the following?
```scss
.c-card__cover {
  /* card cover styles */

  .c-card--horizontal & {
    @include media-query.breakpoint-classes(
      $from: m,
      $to: xl,
      $include-default: false
    ) {
      block-size: 100%; 
      inline-size: 100%; 
    }
  }
}
```

Reply:
@gerardo-rodriguez I recommend using the `breakpoint-parent-classes` mixin instead (see the comments in `_media-query.scss` for explanation and an example), it was meant for use cases like this.

Reply:
Thanks, @tylersticka. I didn't know about the `breakpoint-parent-classes` mixin, it works great! 😁 🎉 

https://github.com/cloudfour/cloudfour.com-patterns/pull/2057/commits/9f1705c64b1e95c732f8c46c019c3da7a881c957